config.yaml

```diff
@@ -16,7 +16,9 @@ googleAnalytics: 'UA-141833521-1'
 enableRobotsTXT: true
 canonifyURLs: true
 
-permalinks: { post: '/blogs/:title/' }
+permalinks:
+    post: '/blogs/:title/'
+    case-studies: '/projects/case-studies/:title'
```

Comment:
Please change the url path to '/projects/case-study/:title'

Reply:
suggested change done in commit [9e1bead](https://github.com/scalereal/website/pull/160/commits/9e1beade7fa0bb2f8fbaccd8fb7cf6fc68e3076d)